public/_redirects

```diff
@@ -1,2 +1,60 @@
+/guides/auth/using-an-access-token.html    /guides/auth/using-an-access-token
+/guides/sandbox-setup/    /guides/sandbox
+/guides/webhooks/obtain-authorization.html    /guides/webhooks
+/guides/webhooks/create-subscription.html    /guides/webhooks/create-subscription
+/guides/webhooks/validating-webhooks.html    /guides/webhooks/validating-webhooks
+/guides/webhooks/process-webhooks.html    /guides/webhooks/process-webhooks
+/resources/personal-verified-customer.html    /guides/personal-verified-customer
+/resources/personal-verified-customer/create-personal-verified-customers.html    /guides/personal-verified-customer/create-a-customer
+/resources/personal-verified-customer/handling-verification-statuses-personal.html    /guides/personal-verified-customer/handle-verification-statuses
+/resources/personal-verified-customer/frequently-asked-questions.html    /guides/personal-verified-customer/frequently-asked-questions
+/resources/business-verified-customer.html    /guides/business-verified-customer
+/resources/business-verified-customer/create-business-verified-customers.html    /guides/business-verified-customer/create-a-customer
+/resources/business-verified-customer/handling-controller-and-customer-statuses.html    /guides/business-verified-customer/handle-verification-statuses
+/resources/business-verified-customer/adding-beneficial-owners.html    /guides/business-verified-customer/add-beneficial-owners
+/resources/business-verified-customer/handling-beneficial-owner-certification.html    /guides/business-verified-customer/certify-beneficial-ownership
+/resources/business-verified-customer/frequently-asked-questions.html    /guides/business-verified-customer/frequently-asked-questions
+/resources/funding-source-verification/micro-deposit-verification.html    /guides/micro-deposit-verification
+/resources/dwolla-js.html    /guides/dwolla-js/add-a-bank-account
+/resources/dwolla-js/add-a-bank-account.html    /guides/dwolla-js/add-a-bank-account
+/resources/funding-source-verification/instant-account-verification.html    /guides/dwolla-js/instant-account-verification
+/resources/dwolla-js/instant-account-verification.html    /guides/dwolla-js/instant-account-verification
+/resources/dwolla-js/on-demand-bank-transfers.html    /guides/dwolla-js/instant-account-verification
+/resources/dwolla-plaid-integration.html    /guides/plaid
+/resources/testing.html    /guides/sandbox
+/resources/    /concepts
+/resources/account-types.html    /concepts/customer-types
+/resources/funding-source-verification.html    /guides/dwolla-js/instant-account-verification
+/resources/balance-funding-source.html    /concepts/balance-funding-source
+/resources/bank-transfer-workflow/processing-times.html    /concepts/transfer-processing-times
+/resources/same-day-ach.html    /concepts/same-day-ach
+https://developers.dwolla.com/resources/bank-transfer-workflow/transfer-failures.html    /concepts/transfer-failures
```

Comment:
Is this a typo with the full URL?

Reply:
AAhhh, yep, sure is! good catch. I'll update